### ISSUE TITLE
[MainUI] Display map for a location (settings)

### ISSUE THREAD

**Issue by lolodomo** [openhab/openhab-webui#411]
In settings, when I open the link to a Location item or when I open a location item, what is displayed at top is a string with the coordinates.
A cool improvment would be to display the position inside a map.

**Comment by lolodomo** [comment]
You could do the same thing as in the regional setting, with this to open the map with the position:
![image](https://user-images.githubusercontent.com/10186704/96908717-4f6e9400-149d-11eb-98c0-a39f90cf1f7d.png)

**Comment by lolodomo** [comment]
I should admit that displaying the position on a map when you are in the settings is not something very useful. 
While we have the ability to create a map view page and to open this page when clicking on the item in model views, I believe my initial request can be closed.

**Comment by ghys** [comment]
No, it would still be useful as a popup I think. It's just that I don't have time to do it.

**Comment by jsjames** [comment]
@florian-h05, since the map picker exists now, we can close this one, correct?

**Comment by florian-h05** [comment]
I do think so 👍
